app/views/schools/add_participants/details.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "Add early career teacher or mentor - participant details" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @school.name %>, <%= @cohort.display_name %> cohort</span>
+    <h1 class="govuk-heading-l">Enter their personal details</h1>
+
+    <%= form_for add_participant_form, url: { action: :update }, method: :post do |f| %>
```

Comment:
action `update` method `post`? That sounds a bit surprising.

Reply:
For some reason cannot respond to precious comment: `I don't think we want to spend time on writing a form wizard.` - I've worked with wicked before and I found fighting with it much more than the time spent on writing this, especially around dynamic step ordering. It was a while ago, so maybe it would be better this way though...